content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+# Listing XRP as an Exchange
+
+This document describes the steps that an exchange needs to take to list XRP. For details about other aspects of `rippled` and the Ripple Consensus Ledger (RCL), see the  [Ripple Developer Center](https://ripple.com/build).
+
+## Alpha Exchange
+
+For illustrative purposes, this document uses a fictitious business called _Alpha Exchange_ to explain the high-level steps required to list XRP. For the purposes of this document, Alpha Exchange:
+
+* Currently specializes in listing BTC/USD
+
+* Wants to add BTC/XRP and XRP/USD trading pairs
+
+* Maintains balances for all of its customers
+
+* Maintains balances for each of its supported currencies
+
+### User Benefits
+
+Alpha Exchange wants to list BTC/XRP and XRP/USD trading pairs partially because listing these pairs will benefit its users. Specifically, this support will allow its users to:
+
+* Deposit XRP _to_ Alpha Exchange _from_ the RCL
+
+* Withdraw XRP _from_ Alpha Exchange _to_ the RCL
+
+* Trade XRP with other currencies, such as BTC, USD, amongst others
+
+## Prerequisites for Supporting XRP
+
+To support XRP, Alpha Exchange must:
+
+* Create and maintain new [accounts](#accounts)
+
+* Create and maintain [balance sheets](#balance-sheets)
+
+See also:
+
+* [Gateway Compliance](https://ripple.com/build/gateway-guide/#gateway-compliance) — Gateways and exchanges are different, but exchanges should also ensure that they are complying with local regulations and reporting to the appropriate agencies.
+
+* [Requirements for Sending to RCL](https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl)
+
+* [Requirements for Receiving from RCL](https://ripple.com/build/gateway-guide/#requirements-for-receiving-from-rcl)
+
+* [Gateway Precautions](https://ripple.com/build/gateway-guide/#precautions)
+
+### Accounts
+
+XRP is held in *accounts* (sometimes referred to as *wallets* ) on the Ripple Consensus Ledger (RCL). Accounts on the RCL are different than accounts on other blockchain ledgers, such as Bitcoin, where accounts incur little to no overhead. To submit transactions (for example, [OfferCreate](https://ripple.com/build/transactions/#offercreate) and others used for trading), RCL accounts require XRP [reserves](https://ripple.com/build/reserves/) to protect the ledger against spam and malicious usage. On other blockchains, balances are derived from the previous block. On the RCL, [account objects](https://ripple.com/build/ledger-format/#accountroot) describe several other properties of the account in addition to balances, so accounts are represented in each ledger and can never be destroyed or removed. Exchanges do not need to create accounts for each customer that holds XRP; they can store all of their customers’ XRP in just a few RCL accounts. For more information about RCL accounts, see the [Accounts](https://ripple.com/build/accounts/) article.
```

Comment:
This paragraph is quite a wall of dense text. It also conflates the XRP transaction cost and the XRP reserve (two related, but different, features). Also, Bitcoin's model doesn't have "accounts" in the same way RCL has "accounts"—it's the Unspent Transaction Outputs (UTXO) that matter. Maybe some of this paragraph can be moved to elsewhere or removed entirely. I think I'd boil it down to the following:

> XRP is held in _accounts_ (sometimes referred to as _wallets_ ) on the Ripple Consensus Ledger (RCL). Accounts on the RCL are different than addresses on other blockchain ledgers, such as Bitcoin, where using new addresses incurs little to no overhead. In the RCL, accounts [can never be deleted](concept-accounts.html#permanence-of-accounts), and each account must hold a separate [reserve of XRP](concept-reserves.html) which cannot be sent to others. For this reason, Ripple encourages institutions not to create new accounts needlessly.